SYCL/ESIMD/api/functional/type_coverage.hpp

```diff
@@ -86,28 +91,42 @@ template <tested_types required> auto get_tested_types() {
   }
 }
 
+auto get_all_vector_lengths() { return values_pack<1, 8, 16, 32>(); }
```

Comment:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/588/commits/b359d36a0cf7e324f477dea39bd6f3736b648c55#diff-3bbc1ab1445c827282cc08d73f76b58cf553d9ced2c799ba46e6148583896d22R94-R104)

Reply:
What do we need the `enum class tested_dims { all };` for?
I mean, when we need some custom values pack, we might just use it explicitly on a test-wide basis.. Am I missing something?

Reply:
to have prepared enum for the future, now I don't know which dimensions could be useful

Reply:
Do you mean something like `single` with `values_pack<8>` for example? Let's add it then, so that test would use the same value for case when all tests should be run on some `single` dimension from the full set)